.github/workflows/twister.yaml

```diff
@@ -135,6 +135,7 @@ jobs:
       fail-fast: false
       matrix:
         subset: ${{fromJSON(needs.twister-build-prep.outputs.subset)}}
+    timeout-minutes: 1440
```

Comment:
Shouldn't this override only apply to the weekly runs and not normal PRs?

Reply:
You are correct in that this override is only _necessary_ for the scheduled (weekly) runs.

We could be more verbose here and specify different timeouts for different run types; but, that is not very trivial without using the infamous GH "fake ternary operator" hack that sometimes works (i.e. using `&&` and `||` as ternary op because GH refuses to add the support for real one ...), which I want to avoid unless it is absolutely necessary.

Moreover, the purpose of this is more or less to disable timeout globally, rather than to set the "correct" timeout value because timing out in itself does not really help remedy any problem.